src/CreepLaw/Data/DislocationCreep.jl

```diff
@@ -748,7 +748,7 @@ function wet_clinopyroxene_Chen_2006()
     data = DislocationCreep(;
         Name="Wet Clinopyroxene | Chen et al. (2006)",
         n=2.7NoUnits,
-        A=(10^6.7)MPa^(-27//10) / s,
+        A=(10^(67//10))MPa^(-27//10) / s,
```

Comment:
The rational number has to be only in the units, the actual value should be still a float. Same elsewhere